### DISCUSSION TITLE
Add an history navigation pair of buttons

### DISCUSSION THREAD

**Discussion by sixdouglas** [Kong/insomnia#6161] [Ideas]
I'm great fan of Insomnia, I'm using it on a big projects with multiple APIs and environments.
But what I miss the most is the ability to go back to the previous request or, once there, to return to the next.

The functionality would be the same as the one in a browser, using a `<` or a `>` pair of buttons.

What do you think of it?

**Comment by NickRimmer** [comment]
Hi, I also missed quick access to my active requests I'm working with. So, I created tabs for this (;
https://insomnia.rest/plugins/insomnia-plugin-request-navigator
